source/templates/pmc-member-welcome.txt

```diff
@@ -0,0 +1,22 @@
+To: private@[PROJECT].apache.org, [PMC member ID]@apache.org
+Subject: Welcome to the PMC, [First Name] 
+
+Hello, [PMC Member],
+
+Welcome to the [Project] Project Management Committee. As a PMC
+member, you now are empowered help make decisions about, and guide the
+future of, our project.
+
+Please subscribe to the [PROJECT] Project Management
+Committee mailing list private@[PROJECT].apache.org. Do this by
+sending an empty email to private-subscribe@[PROJECT].apache.org and
+follwing the instructions in the automated response you'll receive.
```

Comment:
I thought PMC members were automatically subscribed but this is just my memory and not any concrete knowledge of the process so I'm likely wrong.

Reply:
Hmm. Actually not sure. We have ended up with a *lot* of folks, recently, not subscribed to PMC/Private lists, so quite possibly this automation is missing or broken? I think it's harmless to ask them to do it, and it'll tell them they're already subscribed? But I'll check into this.

Reply:
We always asked new PMC members to subscribe - and the option to self-subscribe only appeared in the UI after the person has been added to the PMC group, so I guess it's just "availability to subscribe" that is enabled when the new member is added.